### ISSUE TITLE
Auto completion suggestions are not triggered automatically.

### ISSUE THREAD

**Issue by ADmad** [felixfbecker/php-language-server#263]
For e.g. if I have a class method with argument `$options`, when typing `$op` in the method body i would expect to get completion suggestion list, but i don't any. To trigger the suggestions list I have to use `Ctrl + Space` keys. Is this expected behavior?

I am using the language server release `4.2.2` and VS Code version `1.9.0-insider`
Commit `4265f5e35cb3ea8ba7e0cf6b14ec606b6f8647ea`.

I have set the new vscode settings `php.suggest.basic` and `php.validate.enable` to `false`.

**Comment by felixfbecker** [comment]
`triggerCharacters` is set here: https://github.com/felixfbecker/php-language-server/blob/master/src/LanguageServer.php#L261

The language server cannot control anything else

**Comment by ADmad** [comment]
Sorry I don't understand your answer. Even after typing `$` with cursor placed directly after `$` I don't get any completions list. I have to use `Ctrl + space` to get the completions list.

**Comment by felixfbecker** [comment]
But that is out of control of the language server, it can only provide VS Code with `triggerCharacters`. This must be a bug in VS Code then.

**Comment by ADmad** [comment]
Ah gotcha. So turning off `php.suggest.basic` seems to turn off completion suggestions all together :unamused:
